<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupDirectoryEditDialog</class>
 <widget class="QDialog" name="BackupDirectoryEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Úprava složky</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="btnBackupFolder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Interval zálohování:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="pixmap">
      <pixmap resource="../../res/resources.qrc">:/16/icons8_Date_To_16px.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string> Zrušit</string>
       </property>
       <property name="icon">
        <iconset resource="../../res/resources.qrc">
         <normaloff>:/16/icons8_Delete_16px.png</normaloff>:/16/icons8_Delete_16px.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string> Uložit</string>
       </property>
       <property name="icon">
        <iconset resource="../../res/resources.qrc">
         <normaloff>:/16/icons8_Checkmark_16px.png</normaloff>:/16/icons8_Checkmark_16px.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cmbBackupInterval">
     <item>
      <property name="text">
       <string>5 minut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 minut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30 minut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 hodina</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 hodiny</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 hodin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12 hodin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 den</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7 dní</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14 dní</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 měsíc</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="pixmap">
      <pixmap resource="../../res/resources.qrc">:/16/icons8_Alarm_Clock_16px.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Uchovávat staré verze po:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="cmbBackupKeepInterval">
     <item>
      <property name="text">
       <string>5 minut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 minut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30 minut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 hodina</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 hodiny</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 hodin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12 hodin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 den</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7 dní</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14 dní</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 měsíc</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="pixmap">
      <pixmap resource="../../res/resources.qrc">:/16/icons8_Database_16px.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Zálohovaná složka:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Složka se zálohami:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnSourceFolder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="pixmap">
      <pixmap resource="../../res/resources.qrc">:/16/icons8_Open_16px.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_9">
     <property name="pixmap">
      <pixmap resource="../../res/resources.qrc">:/16/icons8_Private_16px.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_10">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Vynechat tyto položky:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" rowspan="2">
    <widget class="QTextEdit" name="teExcludeFilter">
     <property name="toolTip">
      <string>Použití:
*.tmp (jakýkoli soubor s příponou tmp)
dir/* (složka dir)
a.txt (soubor a.txt)</string>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
